CHANGELOG.md

```diff
@@ -4,7 +4,18 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.6.0 (2022-02-16)
+This release adds major new features since the 2.5.2 release, among
+them:
+
+* Compression in continuous aggregates
+* Experimental support for timezones in continuous aggregates
+* Experimental support for monthly buckets in continuous aggregates
+
+It also includes several bug fixes. Telemetry reports are switched to a new
+format, and now include more detailed statistics on compression, distributed
+hypertables and indexes.  We deem this release to be moderate priority for
+upgrading.
```

Comment:
```suggestion
This release is medium priority for upgrade. We recommend that you upgrade at the next available opportunity.

This release adds major new features since the 2.5.2 release, among
them:

* Compression in continuous aggregates
* Experimental support for timezones in continuous aggregates
* Experimental support for monthly buckets in continuous aggregates

It also includes several bug fixes. Telemetry reports are switched to a new
format, and now include more detailed statistics on compression, distributed
hypertables and indexes.  
```